### PR TITLE
Replace other tutorials with dashboard tutorial

### LINKED ISSUE #29: Remove placeholder icons from try page
# Expected Behavior
Icons on the try page should link to tutorials that exist or ones that will be available very soon

# Actual Behavior
There are many icons that are placeholders and may not be linked to tutorials. This may cause confusion.  For example, the "Connect to Anthos platform" and "Connect to Jenkins/Jenkins X".

# Additional Info
Some of these icons should perhaps be removed for now, and could be added back later if tutorials are added at a later date.

### LINKED ISSUE #32: Add Dashboard Tutorial to the Website
# Expected Behavior
I expect the website's [tutorial page](https://tekton-website-258208.firebaseapp.com/try/) to have a tutorial for using the Dashboard.

Based on a discussion in the Docs & Website meeting, I created a Tekton Dashboard tutorial in Katacoda: https://www.katacoda.com/ncskier/scenarios/tekton-dashboard. The intent of this tutorial is to have the website's `Monitor with Tekton dashboard` tutorial link to my Katacoda tutorial.

Please let me know if there are any updates/improvements I should make to my Katacoda tutorial; for example, to improve consistency across tutorials.

# Actual Behavior
The `Monitor with Tekton dashboard` tutorial does not link to anything.

# Additional Info
FYI
@bobcatfish 
@michaelawyu

### PR DESCRIPTION
**Pull request by bobcatfish**
# Changes

This tackles several issues:
* Until we get to #36 it removes the tutorials that we haven't audited
  and updated
* It fixes #32 by adding the dashboard tutorial
* It fixes #29 by removing the placeholder links (and the link to the
  google specific CI/CD docs which we can revisit later)

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by bobcatfish** [comment]
plz review @anneqm @vdemeester @skaegi

File: layouts/try/baseof.html [hunk 1]
```diff
@@ -21,116 +22,26 @@
         .subtitle-col-custom-style {
             margin-right: 20px;
             padding-bottom: 20px;
-        };
+        }
+
+        ;
```
Comment (sbwsg) [inline layouts/try/baseof.html]: no need for a semicolon outside the CSS block here I don't think.  Not sure why there was one before this PR either. 🤔

Comment (bobcatfish) [reply layouts/try/baseof.html]: whoops!

**Comment by bobcatfish** [comment]
PTAL @sbwsg 🙏

**Pull request merged**